homeassistant/components/camera/raspberry_camera.py

```diff
@@ -0,0 +1,70 @@
+"""Camera platform that has a Raspberry Pi camera."""
+
+import os
+import subprocess
+import logging
+
+from homeassistant.components.camera import Camera
+
+_LOGGER = logging.getLogger(__name__)
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """Setup the Raspberry Camera."""
```

Comment:
Before setting up the camera we should check if `raspistill` is missing. If so, the setup of the platform should fail with an error.